src/styles/components/_docs.scss

```diff
@@ -14,6 +14,15 @@
     min-height: calc(100vh - #{$mobileHeaderH});
     padding: 30px 15px;
   }
+
+  select {
+    -webkit-appearance: none; -moz-appearance: none; -ms-appearance: none; appearance: none;
```

Comment:
The prefixing should be done by webpack.

Reply:
not even needed... anyway just kept the `appearance: none;`.